src/main/scripts/jbang

```diff
@@ -200,7 +199,7 @@ if [[ -z "$JAVA_EXEC" ]]; then
       fi
       mkdir -p "$TDIR/jdks"
       echo "Downloading JDK $javaVersion. Be patient, this can take several minutes..." 1>&2
-      jdkurl="https://api.foojay.io/disco/v2.0/directuris?distro=$distro&javafx_bundled=false&libc_type=$libc_type&archive_type=$file_type&operating_system=$os&package_type=jdk&version=$javaVersion&release_status=$release&architecture=$arch&latest=available"
```

Comment:
why need to remove this ?

Reply:
It's only the `release_status=` that gets removed. If you don't set it it already does what you'd normally want it to do: it will use a GA if it's available and only use an EA when that's the only option. Being able to force an install of an EA over a GA seems like a useless option, especially without being able to exactly specify which version you'd want to install. (But if we ever get that option we'd still not need to use this parameter)